alacritty/src/cli.rs

```diff
@@ -138,8 +140,27 @@ impl Options {
                     .takes_value(true)
                     .help("Override configuration file options [example: cursor.style=Beam]"),
             )
+            .subcommand(
+                SubCommand::with_name("msg").about("Send a message to the Alacritty socket").arg(
+                    Arg::with_name("create-window")
```

Comment:
Yeah I get where you're coming from. I've actually thought about the same thing because "create-window" is more a message you send through `alacritty msg` than an argument that it takes.

I'd definitely say that's worth considering once the other issues are ironed out. Might also be interesting to consider how future messages might be structured and what other parameters we may want to pass. URxvt for example allows passing any CLI parameter through their socket, I think that would make sense for a future iteration.